src/App.js

```diff
@@ -13,18 +13,35 @@ import Mockman from "mockman-js";
 import { useEffect } from "react";
 import "./index.css";
 import { useTheme } from "./context/theme-context";
+import { Toast } from "./components/index";
+import { useToast } from "./context/toast-context";
 
 function App() {
   const { theme } = useTheme();
+  const { toastData, setToastData } = useToast();
+  console.log(toastData);
 
   useEffect(() => {
     localStorage.removeItem("token");
     localStorage.removeItem("username");
     localStorage.removeItem("email");
   }, []);
```

Comment:
Suggestion: Can use localStorage.clear() instead of removing the objects individually.

Reply:
Yaah , thanks for suggesting that, I will do that